docs/reference/deploy/master-config-reference.rst

```diff
@@ -1163,6 +1179,54 @@ those partitions/queues.
    the HPC partition named ``defq_GPU`` with the ``gpu_type`` property set, and Slurm constraint
    associated with the feature ``XL675d`` used to identify the model type of the compute node.
 
+**********************************
+ ``additional_resource_managers``
```

Comment:
I know we don't have this yet but it would be cool to just like walk through an example this since it feels like there is a lot missing from these docs.

Like setting kubeconfig and also setting masterip/port for the different resource managers is missing currently. It would be great to have a example of doing multicloud or multi gke cluster.

Reply:
agreed -- I don't think this is in scope of this specific PR though, but perhaps @tara-det-ai has a better idea of where we can add an example like this?

Reply:
yes agree! let's make this out of scope for this PR but not lose the information. i've set up a draft PR here where we can add anything we like. https://github.com/determined-ai/determined/pull/9025